src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue

```diff
@@ -242,17 +242,13 @@ export default {
       }
     },
     vendorFieldLabel () {
-      return `${this.$t('TRANSACTION.VENDOR_FIELD')} - ${this.$t('VALIDATION.MAX_LENGTH', [64])}`
+      return `${this.$t('TRANSACTION.VENDOR_FIELD')} - ${this.$t('VALIDATION.MAX_LENGTH', [this.session_network.vendorField.maxLength])}`
```

Comment:
You could replace this `this.session_network.vendorField.maxLength` with a computed property `vendorFieldMaxLength`, here and in other places.